public/locales/en/author-guide.json

```diff
@@ -119,13 +119,13 @@
             "p2": "At the point of submission, authors should indicate in their Cover Letter that they want their work to be considered as part of a Special Issue, citing the relevant title as indicated below. Submissions are assessed using the same criteria as other eLife submissions. Authors can submit using the Research Articles, Short Reports, Tools and Resources and Research Advances article types. The eLife editors may wish to commission Review Articles as part of a Special Issue.",
```

Comment:
should this line not start: `At the point of submission, authors should include a Cover Letter with their submission,`

Reply:
It should yes and it did.... Will redo